contributing/documentation/standards.rst

```diff
@@ -160,6 +172,7 @@ English Language Standards
 .. _`the Sphinx documentation`: http://sphinx-doc.org/rest.html#source-code
 .. _`Twig Coding Standards`: http://twig.sensiolabs.org/doc/coding_standards.html
 .. _`American English`: http://en.wikipedia.org/wiki/American_English
+.. _`American English Oxford Dictionary`: http://www.oxforddictionaries.com/
```

Comment:
We should link to http://www.oxforddictionaries.com/definition/american_english/ to set the language to ENG (US).